### PR TITLE
Clean up various Python version related things

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,8 @@ python:
   - "3.7"
   - "3.8"
   - "3.9"
-  - "3.10-dev"
+  - "3.10"
+  - "3.11-dev"
```
Comment (fabfuel) [inline .travis.yml:10]: Hey Paul!

Seems like Python `3.10` and `3.11-dev` are only available in Ubuntu 18.04 and 20.04. 
Could you try adding this to the root of the file:
```
dist: bionic
```
sources: 
- https://docs.travis-ci.com/user/languages/python/#python-versions
- https://docs.travis-ci.com/user/reference/bionic/

**Comment by pabs3** [comment]
Updated the commit that switches from 3.10-dev to 3.10 to use bionic.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by fabfuel** [comment]
I also could not make Travis work with Python 3.11.
I have replaced the Travis CI integration with GitHub Actions, which now also runs test for Python 3.11 (alpha 5). Thanks for your PR anyways @pabs3 !

**Comment by pabs3** [comment]
@fabfuel Fair enough re GitHub Actions.

My pull request included one commit changing requirements_dev.txt that
should be applied regardless of which CI solution is used. I've updated
it to drop the other requests, would you mind reopening and merging?

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by fabfuel** [comment]
Hi Paul, right, I overlooked that change.
You have not updated your branch yet, right? If you do, sure, I'll merge it. But I can add the change on my own as well, let me know what you prefer.

**Comment by fabfuel** [comment]
Ah! You did change it already, but GitHub does not show the update and I can not reopen the PR. 

Bildschirmfoto 2022-03-13 um 16 41 49


Would you create a new PR? I will merge asap 🙂

**Pull request closed without merging**